### ISSUE TITLE
Mailserver not available popup shows up when mailserver available

### ISSUE THREAD

**Issue by PascalPrecht** [status-im/status-desktop#7332]
A while back we've introduced a popup that shows up when we get a "mailserver not working" signal from status-go:

![Screenshot from 2022-09-13 08-44-47](https://user-images.githubusercontent.com/445106/189829419-61d3cd97-e527-433a-8963-19117b540d4b.png)

For some reason, this is now shown when there's a mailserver **available** signal.
The exact opposite of what this is supposed to do.

**Comment by richard-ramos** [comment]
@PascalPrecht this popup is shown when the MailserverNotWorking signal is emitted. The thing is that the mailserver cycle might have done a loop and connected to a different mailserver, hence why the popup is shown while the MailserverAvailable signal is emitted.

IMO a popup is certainly not the right UI element to show that a mailserver is not available, since it disrupts the normal user flow and most of the time it would not require an action from the user due to the way the mailserver cycle works. (In fact, Retry button does nothing at all except closing the popup). Ideally we should have a status bar or maybe a toast message to indicate that something is going on with the mailservers and the user could chose whether to act on it or not

**Comment by PascalPrecht** [comment]
Fully agree. I think there has been discussions around this to show an online/offline indicator /cc @igor-sirotin @caybro @alexandraB99

**Comment by caybro** [comment]
Yup, being discussed atm in https://github.com/status-im/status-desktop/pull/7748

**Comment by richard-ramos** [comment]
Unassigning myself as this issue will be solved with UI work either in #7748 or #6213
